### PR TITLE
core(screenshots): align filmstrip to observed metrics

### PR DESCRIPTION
**Pull request by patrickhulce**
followup fix as promised in #4951 

(easier with https://github.com/GoogleChrome/lighthouse/pull/4965/files?w=1)

### CODE REVIEW

File: lighthouse-core/audits/screenshot-thumbnails.js [hunk 1]
```diff
@@ -67,62 +66,70 @@ class ScreenshotThumbnails extends Audit {
...
-          timestamp: targetTimestamp * 1000,
-          data: base64Data,
+    const speedline = await artifacts.requestSpeedline(trace);
+
+    let alternativeMaximumValue = 0;
+    if (context.settings.throttlingMethod !== 'simulate') {
+      const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
+      const metricComputationData = {trace, devtoolsLog, settings: context.settings};
+      const ttci = artifacts.requestConsistentlyInteractive(metricComputationData);
+      alternativeMaximumValue = (await ttci.catch(() => ({timing: 0}))).timing;
```
Comment (brendankenny) [inline lighthouse-core/audits/screenshot-thumbnails.js]: ha, I'm not sure I'm wild about this approach. try/catch is ugly, but this makes it look like it's doing something much more complicated than it is :S

File: lighthouse-core/audits/screenshot-thumbnails.js [hunk 2]
```diff
@@ -67,62 +66,70 @@ class ScreenshotThumbnails extends Audit {
...
-          jpeg.encode(thumbnailImageData, 90).data.toString('base64');
-
-        cachedThumbnails.set(frameForTimestamp, base64Data);
-        thumbnails.push({
-          timing: Math.round(targetTimestamp - speedline.beginning),
-          timestamp: targetTimestamp * 1000,
-          data: base64Data,
+    const speedline = await artifacts.requestSpeedline(trace);
+
+    let alternativeMaximumValue = 0;
```
Comment (brendankenny) [inline lighthouse-core/audits/screenshot-thumbnails.js]: add a comment why there needs to be an `alternativeMaximumValue`? (there's a comment below but maybe some variant should be up here so you don't spend too long thinking about it :)

File: lighthouse-core/audits/screenshot-thumbnails.js (hunk 1 above)
Comment (patrickhulce) [reply lighthouse-core/audits/screenshot-thumbnails.js]: fixed :P

File: lighthouse-core/audits/screenshot-thumbnails.js (hunk 2 above)
Comment (patrickhulce) [reply lighthouse-core/audits/screenshot-thumbnails.js]: yeah good call, done

**Pull request merged**